packages/core/content-manager/admin/src/pages/ListView/components/BulkActions/Actions.tsx

```diff
@@ -292,195 +298,146 @@ const BulkActionModal = ({
-//         Contracts.CollectionTypes.BulkUnpublish.Request['body']
-//       >(`/content-manager/collection-types/${slug}/actions/bulkUnpublish`, body);
-
-//       return data;
-//     },
-//     {
-//       onSuccess() {
-//         toggleNotification({
-//           type: 'success',
-//           message: {
-//             id: 'content-manager.success.record.unpublish',
-//             defaultMessage: 'Unpublished',
-//           },
-//         });
-
-//         queryClient.invalidateQueries(['content-manager', 'collection-types', slug]);
-//         setSelectedEntries([]);
-//       },
-//       onError(error) {
-//         toggleNotification({
-//           type: 'warning',
-//           message: formatAPIError(error),
-//         });
-//       },
-//     }
-//   );
-
-//   const handleConfirmUnpublishAllData = async () => {
-//     await bulkUnpublishMutation.mutateAsync({ ids });
-//   };
-
-//   const showUnpublishButton =
-//     hasPublishPermission && selectedEntriesObjects.some((entry) => entry.publishedAt);
-
-//   if (!showUnpublishButton) return null;
-
-//   return {
-//     actionType: 'unpublish',
-//     variant: 'tertiary',
-//     label: formatMessage({ id: 'app.utils.unpublish', defaultMessage: 'Unpublish' }),
-//     dialog: {
-//       type: 'dialog',
-//       title: formatMessage({
-//         id: 'app.components.ConfirmDialog.title',
-//         defaultMessage: 'Confirmation',
-//       }),
-//       content: (
-//         <Flex direction="column" alignItems="stretch" gap={2}>
-//           <Typography id="confirm-description" textAlign="center">
-//             {formatMessage({
-//               id: 'popUpWarning.bodyMessage.contentType.unpublish.all',
-//               defaultMessage: 'Are you sure you want to unpublish these entries?',
-//             })}
-//           </Typography>
-//           {hasI18nEnabled && (
-//             <Box textAlign="center" padding={3}>
-//               <Typography textColor="danger500">
-//                 {formatMessage(
-//                   {
-//                     id: getTranslation('Settings.list.actions.unpublishAdditionalInfos'),
-//                     defaultMessage:
-//                       'This will unpublish the active locale versions <em>(from Internationalization)</em>',
-//                   },
-//                   {
-//                     em: Emphasis,
-//                   }
-//                 )}
-//               </Typography>
-//             </Box>
-//           )}
-//         </Flex>
-//       ),
-//       confirmButton: formatMessage({
-//         id: 'app.utils.unpublish',
-//         defaultMessage: 'Unpublish',
-//       }),
-//       onConfirm: handleConfirmUnpublishAllData,
-//     },
-//   };
-// };
+const DeleteAction: BulkActionComponent = ({ documents, model }) => {
+  const { formatMessage } = useIntl();
+  const { schema: contentType } = useDoc();
+  const { selectRow } = useTable('deleteAction', (state) => state);
```

Comment:
```suggestion
  const selectRow = useTable('DeleteAction', (state) => state.selectRow);
```

Try and select just what you want, the selector calculates when it needs to re-render so it's just a bit better for performance